book/src/run_a_node.md

```diff
@@ -0,0 +1,169 @@
+# Run a Node
+
+This document provides detail for users who want to run a Lighthouse beacon node.
+You should be finished with one [Installation](./installation.md) of your choice to continue with the following steps:
+
+1. Enable [checkpoint sync](#step-1-checkpoint-sync);
+1. Set up an [execution node](#step-2-set-up-an-execution-node);
```

Comment:
I'm torn as to whether we should swap the checkpoint sync and execution node sections, seeing as we talk about copying a URL to the clipboard.